.github/maven_prepare.yml

```diff
@@ -0,0 +1,29 @@
+#############################################################################
+# GitHub Action to Run Maven Release Prepare.
+#
+#############################################################################
+name: "Maven Release Prepare"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New Version'
+        required: true
+jobs:
+  prepare:
+    name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run Shell Script
+        run: |
+          ./.ci/maven-prepare.sh ${{ github.event.inputs.version }}
+      - name: Commit and Push
+        run: |
+              git config --global user.name 'github-actions[bot]'
+              git config --global user.email 'github-actions[bot]@users.noreply.github.com'
```

Comment:
Why twice?
In script and here?

Make such set before script execution, so you will not need to bother about it in script at all

Reply:
Changes Done

Reply:
Changes Done